docs/sql-data-sources-jdbc.md

```diff
@@ -156,6 +156,20 @@ the following case-insensitive options:
      </td>
   </tr>
 
+  <tr>
+     <td><code>preActions</code></td>
+     <td>
+       You can specify custom queries which you want to run before reading data from JDBC or writing data to JDBC. It is called per DataFrame, not per session.
+     </td>
+  </tr>
+
+  <tr>
+     <td><code>postActions</code></td>
+     <td>
+        This is a JDBC writer related option. You can specify custom queries which you want to run after writing data to JDBC. It is called per DataFrame, not per session.
```

Comment:
@moomindani Same question for post action.

Reply:
In the initial pull-request, we only supported only one query. However, I added support for multiple queries separated by semicolon.